### ISSUE TITLE
The "Delete" button in userfile "Show" page doesn't ask for confirmation.

### ISSUE THREAD

**Issue by prioux** [aces/cbrain#791]
It should show a quick javascript confirmation requester first...

**Comment by prioux** [comment]
This happens to a lot of forms, no just this one. And also, some delete buttons don't work at all.

I suspect broken javascript... :-(

**Comment by prioux** [comment]
All `link_to` with a `:confirm` fail; transforming it into a `:'data-confirm'` manually works.

The Project's page, in button layout, has little 'X' icons on the lower right corner of some projects, and it doesn't do anything anymore.

**Comment by prioux** [comment]
Related: you can delete tool_configs even when there are tasks that are using them. The task objects keep the value of the delete object. Mayb nullify it, or better prevent removing such TCs.

**Comment by prioux** [comment]
OK I'm close to a fix.

1)

```ruby
# Changed all:

link_to blah blah, :confirm => 'Are you sure'

# to

link_to blah blah, :data => { :confirm => 'Are you sure' }
```

2)

The button view for the project had a CSS resize option when the button was active, changing the position of the "X" link for deleting the project, so we could never click on it. I removed the CSS rule.

3) 

The `hijacker_submit_button` code never put the confirm message anywhere, now I copy it in the dynamic submit_button before it's synthetically clicked. HOWEVER the action is now no longer sent, TOFIX.

**Comment by prioux** [comment]
I'm going to remove completely the custom cbrain confirm interface created by Ehsan.

**Comment by prioux** [comment]
Fixed in #809
